### ISSUE TITLE
The result under L=4 is different

### ISSUE THREAD

**Issue by Jasonare** [yoonsanghyu/Dual-Path-Transformer-Network-PyTorch#3]
Your 'data.py' is a little different from the official code. I think your result is under L=2 actually. My validation loss under L=4 is only -19.004.

**Comment by 2922733627** [comment]
请问楼主是跑的100轮次吗？得出来-19.004

**Comment by 2922733627** [comment]
请问楼主是跑的100轮次吗？得出来-19.004
